packages/terra-worklist-data-grid/src/subcomponents/Cell.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import * as KeyCode from 'keycode-js';
+import '../_elementPolyfill';
+import { injectIntl } from 'react-intl';
+import classNames from 'classnames/bind';
+import ThemeContext from 'terra-theme-context';
+import WorklistDataGridUtils from '../utils/WorklistDataGridUtils';
+import styles from './Cell.module.scss';
+import cellShape from '../proptypes/cellShape';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * String identifier of the row in which the Cell will be rendered.
+   */
+  rowId: PropTypes.string.isRequired,
+  /**
+   * String identifier of the column in which the Cell will be rendered.
+   */
+  columnId: PropTypes.string.isRequired,
+  /**
+   * The cell's row position in the grid. This is zero based.
+   */
+  rowIndex: PropTypes.number,
+  /**
+   * The cell's column position in the grid. This is zero based.
+   */
+  columnIndex: PropTypes.number,
+  /**
+   * The information about the cell.
+   */
+  cell: cellShape.isRequired,
+  /**
+   * Boolean value to indicate if the cell is the tab stop on the grid. At any given time, the grid has only one tab stop.
+   */
+  isTabStop: PropTypes.bool,
+  /**
+   * Boolean indicating whether the Cell is currently selected.
+   */
+  isSelected: PropTypes.bool,
+  /**
+   * String that labels the cell for accessibility.
+   */
+  ariaLabel: PropTypes.string,
+  /**
+   * Boolean indicating that the cell is a row header
+   */
+  isRowHeader: PropTypes.bool,
+  /**
+   * Boolean indicating whether the cell is a row selection cell.
+   */
+  isRowSelectionCell: PropTypes.bool,
```

Comment:
Why is this prop needed if RowSelectionCell is a separate component? The logic for both these components should not cross over; they should either have a parent-child relationship, a single component or be isolated components.

Reply:
The RowSelectionCell is just a wrapper around the Cell. It just provide special content for the Cell and has no behavior of its own. It is the Cell that has the logic. We can push the content of the RowSelectionCell into the Cell and get rid of the RowSelectionCell but felt this separation made it more obvious. Let me know if you want us to do that.

Reply:
removed